server/kapacitors.go

```diff
@@ -299,10 +312,40 @@ func (s *Service) UpdateKapacitor(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if activateKapacitor {
+		// make sure that there is at most one active kapacitor
+		err := s.activateKapacitor(ctx, srcID, id)
+		if err != nil {
+			Error(w, http.StatusInternalServerError, err.Error(), s.Logger)
+			return
+		}
+	}
+
 	res := newKapacitor(srv)
 	encodeJSON(w, http.StatusOK, res, s.Logger)
 }
 
+// activateKapacitor deactivates all other kapacitors excluding the one with supplied ID
+func (s *Service) activateKapacitor(ctx context.Context, srcID int, ID int) error {
+	serversStore := s.Store.Servers(ctx)
+	mrSrvs, err := serversStore.All(ctx)
+	if err != nil {
+		return errors.New("Error loading kapacitors for deactivation")
```

Comment:
Nit: when returning errors, [it is preferred](https://github.com/golang/go/wiki/CodeReviewComments#error-strings) they don't begin with a capital letter.

Reply:
good to know, changed, thx